### ISSUE TITLE
[P - 00:09:46] Retrying request https://pgorelease.nianticlabs.com/plfe/268/rpc

### ISSUE THREAD

**Issue by itzJOKINHA** [finndev/PokeBuddy#403]
help?

**Comment by Ander0128** [comment]
I got the same thing:

https://pgorelease.nianticlabs.com/plfe/113/rpc

**Comment by matusalemme** [comment]
The and....i think at 01:05 Italy time

**Comment by matusalemme** [comment]
the Niantic have reprogrammed her game again

**Comment by matusalemme** [comment]
this new version of pokebuddy have work for 9 hour only....sorry guys....but the niantic no wanna let us win easy.....hehehehe

**Comment by RawPassion** [comment]
working again

**Comment by curtiola** [comment]
No, others are running it with no problems right now, not sure what the problem is though lol

**Comment by RawPassion** [comment]
[P - 16:15:28] [Catch Pokemon] We caught a Horsea (Probability: 37.43%) with CP 310, used 1 x ItemPokeBall and received 210 XP

**Comment by RawPassion** [comment]
probably a little hick up

**Comment by Ander0128** [comment]
I'm just concerned that it flagged us for something.

**Comment by Dropholion** [comment]
I'm getting it too.

**Comment by itzJOKINHA** [comment]
Pokefarmer down too.... ohh fuck

**Comment by itzJOKINHA** [comment]
PLEASE,  NOT AGAIN

**Comment by Ander0128** [comment]
My bot seems to work normal again, however I couldn't seem to log in for about 5 min. All seems well now tho.

**Comment by itzJOKINHA** [comment]
My bot cant catch pokemons , or farm pokestops :( This dont sounds good

**Comment by RawPassion** [comment]
Something seems fishy >.> ima stop for now hehe

**Comment by PromethiumFire** [comment]
Press stop then start. Seems to work again. I think Niantic, Inc are up to something.

**Comment by Jaso36** [comment]
Using RPC Endpoint: pgorelease.nianticlabs.com/plfe/402 comment faire je suis en France dans le centre 
how I am in France in the center
